roles/openshift_logging/templates/es.j2

```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
+            exec:
+              command:
+              - "/usr/share/elasticsearch/probe/readiness.sh"
+            initialDelaySeconds: 5
+            timeoutSeconds: 5
+            periodSeconds: 5
```

Comment:
This does not appear to be a configuration option for readiness probes:
https://docs.openshift.org/latest/dev_guide/application_health.html